operators/hazelcast-platform-operator/5.0.0/metadata/annotations.yaml

```diff
@@ -0,0 +1,16 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: hazelcast-platform-operator
+  operators.operatorframework.io.bundle.channels.v1: alpha
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.13.0+git
+  operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
+  operators.operatorframework.io.metrics.project_layout: go.kubebuilder.io/v3
+
+  # Annotations for testing.
+  operators.operatorframework.io.test.mediatype.v1: scorecard+v1
```

Comment:
@devOpsHelm please remove this 2 lines if you don't have tests

Reply:
Hi @mvalarh, I changed it as you suggested but the following test fails `TASK [deploy_olm_operator_upstream_bundle : Wait for the operator hazelcast-platform-controller-manager pod to start up]`. Any idea how can I get more information?